CASC/PortfolioMode.cpp

```diff
@@ -608,17 +608,15 @@ void PortfolioMode::runSlice(std::string sliceCode, int timeLimitInDeciseconds)
 /**
  * Run a slice given by its options
  */
-void PortfolioMode::runSlice(Options& strategyOpt)
+void PortfolioMode::runSlice(Options& opt)
 {
   System::registerForSIGHUPOnParentDeath();
   UIHelper::portfolioParent=false;
 
-  Options opt = strategyOpt;
   //we have already performed the normalization (or don't care about it)
   opt.setNormalize(false);
   opt.setForcedOptionValues();
   opt.checkGlobalOptionConstraints();
-  *env.options = opt; //just temporarily until we get rid of dependencies on env.options in solving
```

Comment:
Good, I think we should admit defeat here, `env.options` isn't going away any time soon.